tests/test_constraints.py

```diff
@@ -130,13 +131,9 @@ def global_constraints():
         -  AllDifferent, AllEqual, Circuit,  Minimum, Maximum, Element,
            Xor, Cumulative
     """
-    if SOLVERNAME is None:
+    if SOLVERNAME is None or SOLVERNAME in EXCLUDE_GLOBAL:
```

Comment:
Why add the second part of the condition? Now global constraints are not tested for any solver?
I think the previous version was better.

Reply:
yes, I made a mistake here, has since been fixed in master and merged